### PR TITLE
Add TellAboutScreen and Redux Global Variables

### PR DESCRIPTION
**Pull request by ShadowEnchanter**
# Description

What did you implemented/Why did you implemented this:

- Implement-Redux-Data-User-Employ-#25
- Implement Redux for navigation data employ
- Add Screen "TellAboutYouScreens" and components
- Modified Styles "ButtonNext"
- Install package Firebase Storage for Upload Images

### CODE REVIEW

File: ContratApp/src/components/JobDescriptionTextarea/JobDescriptionTextarea.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+//   const dispatch = useDispatch();
+
+//   const {employDescription} = useSelector(
+//     (state: RootState) => state.data,
+//   );
+
+//   return (
+//     <View style={styles.container}>
+//         <Text style={styles.nameInput}>Describe tu oficio y experiencia</Text>
```
Comment (ecruz17) [inline ContratApp/src/components/JobDescriptionTextarea/JobDescriptionTextarea.tsx]: get rid of this pls

File: ContratApp/src/components/JobDescriptionTextarea/style.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+//     container:{
+//         margin: 20,
+//         marginVertical: 10,
+//     },
+//     nameInput: {
+//         fontSize: 16, 
+//         color: '#676E76',
+//         fontWeight: '500',
+//     },
+// });
```
Comment (ecruz17) [inline ContratApp/src/components/JobDescriptionTextarea/style.tsx]: no comments 👀

File: ContratApp/src/components/PriceRangeInputs/style.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+// export const styles = StyleSheet.create({
+//     input: {
+//         color: 'black',
+//         fontSize: 16,
+//         fontWeight: '500',
+//         height: 35,
+//         paddingLeft: 34,
+//         borderWidth: 2,
+//         borderColor: '#D9D9D9',
```
Comment (ecruz17) [inline ContratApp/src/components/PriceRangeInputs/style.tsx]: 👀

File: ContratApp/src/components/UploadImage/style.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import {StyleSheet} from 'react-native';
+
+
+export const styles = StyleSheet.create({
+    container: {
+        // position: 'absolute',
+        // bottom: 10,
+        alignSelf: 'center',
```
Comment (ecruz17) [inline ContratApp/src/components/UploadImage/style.tsx]: pls comments

File: ContratApp/src/components/UploadPicProfile/style.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+    container: {
+        // position: 'absolute',
+        // bottom: 10,
```
Comment (ecruz17) [inline ContratApp/src/components/UploadPicProfile/style.tsx]: these ones too

File: ContratApp/src/components/WorkScheduleInputs/WorkScheduleInputs.tsx [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+//   hintHour: string;
+//   titleHourPM: string;
+//   hintHourPM: string;
+//   value: string;
+//   handleChange: (text: string) => void;
+//   isShowError?: boolean;
+//   errorMessage?: string;
+//   width: number;
+//   isPassword?: boolean;
+// }
```
Comment (ecruz17) [inline ContratApp/src/components/WorkScheduleInputs/WorkScheduleInputs.tsx]: it's a bad practice to leave whole components commented

File: ContratApp/src/components/WorkScheduleInputs/style.tsx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+//         borderColor: '#D9D9D9',
+//         borderRadius: 10,
+//         padding: 2,
+//         paddingHorizontal: 90,
+//         marginRight: 10, // Espacio entre los inputs
+//     },
+//     column: {
+//         marginVertical: 10,
+//         marginLeft: 10,
+//         marginRight: 140,
```
Comment (ecruz17) [inline ContratApp/src/components/WorkScheduleInputs/style.tsx]: 👀

File: ContratApp/src/screens/FormEmployeeScreen/TellAboutYouScreen/style.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    //     flex: 1,
+    //     justifyContent: 'center',
+    //     alignItems: 'center',
+    // },
+    // rowHour: {
+        
+    //     flexDirection: 'row', // Coloca los elementos en fila horizontal
+    //     alignItems: 'center', // Alinea verticalmente los elementos en el centro
+    // },
+});
```
Comment (ecruz17) [inline ContratApp/src/screens/FormEmployeeScreen/TellAboutYouScreen/style.tsx]: check these comments

**Review by ecruz17 — CHANGES_REQUESTED** [review]
it's all right, I think you forgot to add the redux variables to the types.ts file at the src/types folder ;)

File: ContratApp/src/components/WorkScheduleInputs/WorkScheduleInputs.tsx (hunk 6 above)
Comment (ShadowEnchanter) [reply ContratApp/src/components/WorkScheduleInputs/WorkScheduleInputs.tsx]: It's true, thanks for the observation.  🙌

**Review by ecruz17 — APPROVED** [review]
great work!

**Review by AMCernas — APPROVED** [review]

**Pull request merged**